src/routes/console/project-[project]/storage/bucket-[bucket]/create-file/create.svelte

```diff
@@ -29,7 +29,7 @@
 
     async function create() {
         try {
-            uploader.uploadFile(
+            const uploadPromise = uploader.uploadFile(
```

Comment:
we can await here directly
```suggestion
            await uploader.uploadFile(
```

Reply:
Using `await` will keep the wizard open until the upload finishes. Not good in case of medium to large files.